senza/cli.py

```diff
@@ -398,6 +399,18 @@ def parse_args(input, region, version, parameter, account_info):
     return args
 
 
+def read_parameter_file(parameter_file):
+    paras = []
+    try:
+        with open(parameter_file, 'r') as ymlfile:
+            cfg = yaml.load(ymlfile)
+        for key, val in cfg.items():
+            paras.append("%s=%s" % (key, val))
+    except:
```

Comment:
This exception is too broad and can catch unintended Exceptions (like SystemExit and KeyboardInterrupt).